src/index.js

```diff
@@ -59,28 +56,28 @@ app.use((msg) => {
   }
   messageAt = Date.now();
   // "Removes" the COMMAND_PREFIX from the user's message
-  const command = msg.content.slice(COMMAND_PREFIX.length);
+  const command = msg.content.slice(COMMAND_PREFIX.length).split(' ');
```

Comment:
Can destructure those two, then they have more meaningful names that `command[0]` and `command[1]`
```suggestion
  const [command, arg] = msg.content.slice(COMMAND_PREFIX.length).split(' ');
```